tensorflow/python/debug/cli/command_parser.py

```diff
@@ -268,10 +268,10 @@ def parse_ranges(range_string):
     if len(item) != 2:
       raise ValueError("Incorrect number of elements in range")
     elif not isinstance(item[0], (int, float)):
-      raise ValueError("Incorrect type in the 1st element of range: %s" %
+      raise TypeError("Incorrect type in the 1st element of range: %s" %
```

Comment:
Lets keep these as ValueError? Or we'll have to update the Raises: section as well. Typically in TF we just throw ValueError for any input validation problems.

Reply:
Oh, I see, I was just following the Python documentation :

> ValueError
> 
> Raised when an operation or function receives an argument **that has the right type** but an **inappropriate value**, and the situation is not described by a more precise exception such as [IndexError](https://docs.python.org/3/library/exceptions.html#IndexError).